src/scripts/model/round.js

```diff
@@ -0,0 +1,104 @@
+import board from '../board';
+import Answer from './answer';
+import Team from './team';
+import Game from './game';
+import ROUND_STATUS from '../roundStatus';
+/**
+ * Manage current question and board
+ */
+export default class Round {
+
+    constructor(question) {
+        this.right = 0;
+        this.status = ROUND_STATUS.DEFAULT;
+        this.points = 0;
+        this.question = question;
+        board.setQuestion(this.question.getName());
```

Comment:
this breaks MVC rule - Round object shouldn't set question in view.
The best option is use a another class (like RoundView?) to draw this